vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1520,6 +1520,54 @@ public void onRejectInvitation(final String roomId, final SimpleApiCallback<Void
         }
     }
 
+    public void onJoinRoom(final String roomId, final SimpleApiCallback<Void> onSuccessCallback) {
```

Comment:
The code of this method should be the same as the current join define in VectorRoomActivity
https://github.com/vector-im/riot-android/blob/develop/vector/src/main/java/im/vector/activity/VectorRoomActivity.java#L3408